app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt

```diff
@@ -1744,11 +1812,72 @@ class ConversationActivityV2 : PassphraseRequiredActionBarActivity(), InputBarDe
         }
     }
 
+    // Method to check if we have network access and a Session Node path
+    private fun networkAndSessionNodePathIsValid(): Boolean {
+
+        // Check that we have any network connection at all
+        val connectivityManager = getSystemService(CONNECTIVITY_SERVICE) as ConnectivityManager
```

Comment:
There's a ` fun Context.isConnected() : Boolean {` already in the app. Is that method valid?
Is it worth updating that and making it something that can be reused throughout the app?

Reply:
Good eyes! I wasn't aware of that - have adjusted to use & tested - all seems fine. 
Changes made in commit:
https://github.com/oxen-io/session-android/pull/1523/commits/fea66bc770dc65257d4c7624821f3ba96d2522a8